homework3/hw3/Views/LeagueView.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  LeagueView.swift
+//  hw2
+//
+//  Created by Tina Jureško on 14.03.2025..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class LeagueView: BaseView {
+    
+    private let countryLabel = UILabel()
+    private let nameLabel = UILabel()
+    private let logoImageView = UIImageView()
+    private let arrowImageView = UIImageView()
+        
+    override func addViews() {
+        super.addViews()
+        addSubview(logoImageView)
+        addSubview(countryLabel)
+        addSubview(arrowImageView)
+        addSubview(nameLabel)
+    }
+
+    override func styleViews() {
+        arrowImageView.image = UIImage(named: "Vector")
+        logoImageView.contentMode = .scaleAspectFit
+        arrowImageView.contentMode = .scaleAspectFit
+        
+        countryLabel.font = .regularBold14
+        countryLabel.textColor = .primaryBlack
+        countryLabel.numberOfLines = 2
```

Comment:
I na drugim mjestima ovaj numberOfLines = 2? To si upisivala duži text pa si zaključila da ti je tako najbolje?